workspaces/azure-resources/plugins/backstage-azure-resources-backend/README.md

```diff
@@ -0,0 +1,57 @@
+# Azure resources backend plugin
+
+The backend API plugin used by the frontend plugin. This plugin wraps a few Azure Resource Graph queries to pull information about your Azure resource in relation to a backstage entity.
+
+## Getting started
+
+Add following to your backstage instance
+
+```
+# From the Backstage root directory
+cd packages/backend
+yarn add @backstage-community/plugin-azure-resources-backend
+```
```

Comment:
```suggestion
```bash
# From the Backstage root directory
yarn --cwd packages/backend add @backstage-community/plugin-azure-resources-backend
```
```